resources/windows/wix/properties.wxs.erb

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="windows-1252"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+
+    <ComponentGroup Id="Include_Properties" />
+
+    <!-- Add/Remove Programs icon and help link -->
+    <Icon Id="$(var.OurCompanyNameWord).ico" SourceFile="conf\windows\stage\misc\puppetlabs.ico"/>
```

Comment:
Yes - we should change this to a template.
The <icon> element should perhaps not live here - need to figure out best place to put it.